### PR TITLE
[mongodb-replicaset] replace MONGODB_URL with --mongodb.uri

### PR DESCRIPTION
**Pull request by tetianakravchenko**
#### What this PR does / why we need it: 
- mongodb-exporter  doesn't work with env variable `MONGODB_URL` after this PR: https://github.com/percona/mongodb_exporter/pull/116. It should be changed to `-mongodb.uri`
- minor test fixes as follow up to https://github.com/helm/charts/pull/9907

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/mongodb-replicaset/templates/mongodb-statefulset.yaml [hunk 1]
```diff
@@ -229,12 +229,12 @@ spec:
             - sh
             - -ec
             - >-
+              /bin/mongodb_exporter
             {{- if .Values.auth.enabled }}
-              export MONGODB_URL=mongodb://$METRICS_USER:$METRICS_PASSWORD@localhost:{{ .Values.port }};
```
Comment (steven-sheehy) [inline stable/mongodb-replicaset/templates/mongodb-statefulset.yaml:233]: Why the change to connect to admin database for auth enabled? Think it should be as before.

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

File: stable/mongodb-replicaset/templates/mongodb-statefulset.yaml (hunk 1 above)
Comment (tetianakravchenko) [reply stable/mongodb-replicaset/templates/mongodb-statefulset.yaml:233]: done https://github.com/helm/charts/pull/10026/commits/429615119febde083ea72f182d71d9467182c5e0
seems that `admin` is default one

**Review by steven-sheehy — CHANGES_REQUESTED** [review]
There's still a MONGODB_URL in the probe

**Comment by unguiculus** [comment]
It's still possible to use an env var. I'd suggest to only rename to MONGODB_URI .

**Comment by steven-sheehy** [comment]
@unguiculus It looks like MONGODB_URI change is not actually released yet as last release was 0.6.2 in Sept, while the referenced PR is merged only 29 days ago. The command line parameter has been there for a long time and is more consistent with how we configure the other exporter variables like TLS, in my opinion. 

Is this your understanding, @tetianakravchenko ?

**Comment by tetianakravchenko** [comment]
@steven-sheehy yes, referenced PR is not released yet. In the case of  renaming `MONGODB_URL ` -> `MONGODB_URI` mongodb-exporter will stop working for default mongodb-exporter docker image `ssalaues/mongodb-exporter:0.6.1` https://github.com/helm/charts/blob/master/stable/mongodb-replicaset/values.yaml#L54-L57, `-mongodb.uri` should work for default  image and for newer releases as well

**Comment by steven-sheehy** [comment]
/approve
I'll let you lgtm or not @unguiculus
